fairscale/nn/pipe/pipe.py

```diff
@@ -19,12 +19,12 @@
 
 """The Pipe interface."""
 from collections import OrderedDict
+from dataclasses import dataclass, field
```

Comment:
testing with different conda environments to make sure that I caught all the issues, somehow isort behaves differently it seems and pre-commit agrees, sorry about that